.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish Modin wheel to PyPI
+
+on: push
+
+jobs:
+  build-n-publish:
+    name: Build and publish Modin wheel to PyPI
+    environment: release
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
```

Comment:
Why?

Reply:
We should have tagged wheels, otherwise we might get untagged ones.

Reply:
It is possible the latest commit is not the tagged commit, so I think @YarShev is correct. Additionally, we may want to check out the latest tag to ensure we only upload tagged releases.

It would be good to also have a check that ensures that the release name is correct, so it doesn't upload a bad release.

Reply:
Will it be better to use `fetch-tags:` option?